pkg/promclient/promclient.go

```diff
@@ -26,7 +26,6 @@ import (
 	"github.com/gogo/status"
 	"github.com/pkg/errors"
 	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/pkg/labels"
```

Comment:
Ok, so it looks like we are using both of the names for this package in different places. Some lines, e.g. line 160/159 use the `labels` name for this package whereas other places refer to it as `promlabels`. To make this change work, we need to rename all of the usages to make sure that they all use either `labels` or `promlabels` but not both

Reply:
Done ! Changed promlabels to labels for consistency.